.github/workflows/size.yml

```diff
@@ -16,10 +16,3 @@ jobs:
           pattern: 'dist/!(*.js.map)'
           build-script: build
           repo-token: '${{ secrets.GITHUB_TOKEN }}'
-
-      - name: compressed-size-action
-        uses: preactjs/compressed-size-action@v2
-        with:
-          pattern: 'test/fixtures/**/dist/!(*.map)'
-          build-script: 'test'
-          repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
Test suite already covers this really, we don't need a separate readout for file size bumps in a PR comment.

Also this is not aligning w/ the 'Test' CI action, probably Node version differences that isn't worth trying to hack around